sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -215,11 +219,18 @@ export class ConfirmationOfEnrollmentController {
       );
     }
 
-    await this.applicationService.updateCOEStatus(
-      applicationId,
-      COEStatus.submitted,
-    );
+    const updatedCOEStatus =
+      await this.applicationService.updateApplicationCOEStatus(
+        applicationId,
+        COEStatus.completed,
+        ApplicationStatus.completed,
+      );
 
+    if (updatedCOEStatus.affected === 0) {
+      throw new UnprocessableEntityException(
+        `Confirmation of Enrollment and application status update to completed is failed`,
```

Comment:
Please, do not use string interpolation if there is nothing to be interpolated.

Reply:
Okay cool